src/components/common/Stepper.vue

```diff
@@ -52,6 +72,24 @@ export default class Stepper extends Vue {
   justify-content: center;
 }
 
+.step:hover {
+  cursor: pointer;
+
+  .step__btn {
+    background-color: #1976d2;
+    color: $BCgovInputBG;
+  }
+
+  .step__icon {
+    color: $BCgovInputBG;
+  }
+}
+
+.selected {
+  background-color: #1976d2 !important;
+  color: $BCgovInputBG !important;
```

Comment:
There's something a bit odd here as well. I expect that when I hover over a button, I can tell the difference between that one and the selected one. But in this case, they look the same. Try this and compare with the previous stepper implementation.

Reply:
Definitely! In the other stepper the colour was the same except it had an opacity of .8 on mouseover. Will try that here :)

Reply:
Updated in next commit!